cli/tests/environment-edit.bats

```diff
@@ -104,6 +104,7 @@ EOF
 # ---------------------------------------------------------------------------- #
 
 @test "'flox edit' says to re-activate when hook is modified and environment is active" {
+  skip "FIXME: hangs locally"
```

Comment:
suggestion: make an issue
We will otherwise just forget and have a dead test in our suite

Reply:
also possibly related to https://github.com/flox/flox/actions/runs/7205879662/job/19629631416#step:10:116

Reply:
That one in particular was a legitimate test failure.

Reply:
I'll raise with developer support, otherwise it just breaks my ability to run the test suite.

Reply:
Consider not checking in the skip if it still runs successfully in ci